templates/macros.html

```diff
@@ -126,26 +126,26 @@
             {# If the release isn't a library, then display that warning #}
             {% if not release.is_library -%}
                 <a href="{{ release_url | safe }}" class="pure-menu-link warn" title="{{ release_name }} is not a library">
-                    <i class="fa fa-fw fa-warning"></i> {{ release.version }}
+                    {{ "exclamation-triangle" | fas }} {{ release.version }}
```

Comment:
Actually, a bunch of places seem to have been fixed-width previously and are now not.